.github/workflows/publish.yml

```
@@ -0,0 +1,47 @@
+name: Publish to npm
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'package*.json'
```

Is there any reason this isn't just `package.json`? If only `package-lock` is changed, that's probably bad and should be ignored, right?

```suggestion
      - 'package.json'
```

---

Yeah I suppose you're right there. I added the wildcard thinking "oh both the files can change so we should watch both" but thinking about it yeah, we only need to watch the package.json